packages/mobile/src/components/logo/logo-white.tsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import Svg, { SvgProps, Path } from 'react-native-svg';
+import { useCustomTheme } from '~/hooks/hooks';
+
+const LogoWhite = (props: SvgProps) => {
+  const { colors } = useCustomTheme();
```

Comment:
White is just... white)) We don't need different colors for it

Reply:
fixed ))